.github/labeler.yml

```diff
@@ -111,6 +122,7 @@ CI:
 
 rbd:
   - doc/rbd/**
+  - doc/man/8/rbd*
```

Comment:
Nit: I would just move this up one line so that it remains in lexicographic order.

Reply:
updated and repushed.